source/develop/release-management/features/virt/high-performance-vm.html.md

```diff
@@ -0,0 +1,179 @@
+
+#### **Enable Headless Mode and enable Serial console**
+Displayed in console side-tab of the VM dialog.
+
+For Headless VM handling please see http://www.ovirt.org/develop/release-management/features/virt/headless-vm
+
+#### **Disable all USB devices**
+Displayed in console side-tab of the VM dialog
+
+#### **Disable the Sound Card device**
+Displayed in console side-tab of the VM dialog
+
+#### **Enable Pass-Through Host CPU**
+Displayed in Host side-tab of the VM dialog.
+
+This option can only be enabled when VM live migration is disabled.
+
+#### **Disable VM migration**
+Displayed in 'Host' side-tab of the VM dialog.
+
+The VM cannot be migrated, either automatically or manually for oVirt 4.2. 
+Support migration is planned for oVirt 4.3 . See https://bugzilla.redhat.com/show_bug.cgi?id=1457250.
+ 
+#### **Setting the invtsc CPU flag** 
+If CPU supports it,  the invtsc flag will be added (via VDSM or a hook).
+  <feature policy='require' name='invtsc'/>
+
+#### **Enable IO Threads, Num Of IO Threads = 1**
+Displayed in 'Resource Allocation' side-tab of the VM dialog.
+
+#### **Disable the Memory Balloon Device**
+Displayed in 'Resource Allocation' side-tab of the VM dialog.
+
+This option can also be set manually by the user for the cluster of that VM  (add/edit Cluster dialog->Optimization side-tab). 
+
+#### **Disable High availability (for the pinned host)**
+Displayed in 'High Availability' side-tab of the VM dialog.
+
+Currently High availability is not supported when migration is disabled.
+We may plan to support it with an option for re-running the VM on the same pinned host in oVirt 4.2 and expand the implementation to all cluster hosts as part of the solution for supporting migration in oVirt 4.3.
+
+#### **Disable the Watchdog device**
+Displayed in 'High Availability' side-tab of the VM dialog.
+
+#### **Enable paravirtualized Random Number Generator PCI (virtio-rng) device** 
+Displayed in 'Random Generator' side-tab of the VM dialog.
+
+This option is automatically set using libvirt defaults ("Period duration"=1000 milliseconds) and can be changed by the user before saving.
+
+### List of manual configuration settings/warnings for High Performance VM
+
+#### **Set the CPU Pinning topology**
+Displayed in 'Resource Allocation' side-tab of the VM dialog.
+
+In case the CPU Pinning topology is not set upon VM saving, a recommendation/warning will be displayed in a pop-up and the user will be asked to pin the VM to a host (by selecting the "Start running on: Specific Host" in "Host" side-tab) and verify if VM configuration fits the host configuration (i.e. guest_number_of_cores_per_socket <= host_number_of_cores_per_socket,  guest_threads_per_core <= host_threads_per_core and guest_number_of_sockets <= host_number_of_sockets).
+
+Ideally the CPU pinning should be done automatically and not manually by the user, but in first phase we will not support it.  
+
+#### **Set the I/O Thread pinning topology**
```

Comment:
@sgratch, just please make it clear which way we're currently decided to go (automatic NUMA-aware assignment of 1 thread), everything else looks good to me